ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/BlockchainModule.java

```diff
@@ -61,6 +61,7 @@ MutableWorldView getMutableWorldView(
       final GenesisState genesisState,
       @Named("KeyValueStorageName") final String keyValueStorageName) {
     if ("memory".equals(keyValueStorageName)) {
+      System.out.println("trigger bad coverage");
```

Comment:
we want to remove this line before merging

Reply:
Oh yeah sorry I requested your review and then realised there's one last thing I needed to test still😛